app/controllers/public/speaker/view.js

```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+import { action, computed } from '@ember/object';
+
+export default class extends Controller {
+    @computed('speaker.isAuthenticated', 'authManager.currentUser.email', 'authManager.currentUser.isAdmin', 'model.speakers')
+    get showEditSection() {
+      return this.session.isAuthenticated && (this.authManager.currentUser.isAdmin || this.model.email === this.authManager.currentUser.email);
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403732)